db/init_db.js

```diff
@@ -1,31 +1,131 @@
-// code to build and initialize DB goes here
 const {
   client
   // other db methods 
 } = require('./index');
 
+const { 
+  createProduct 
+} = require('./products');
+
 async function buildTables() {
   try {
     client.connect();
+    
+    console.log('Dropping All Tables...');
+    await client.query(`
+      DROP TABLE IF EXISTS order_products;
+      DROP TABLE IF EXISTS orders;
+      DROP TABLE IF EXISTS users;
+      DROP TABLE IF EXISTS products;
+    `);
+    console.log('Finished dropping tables!');
 
-    // drop tables in correct order
-
-    // build tables in correct order
-
+    console.log('Starting to build tables...');
+    await client.query(`
+    CREATE TABLE products(
+      id SERIAL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL,
+      description TEXT NOT NULL,
+      price DECIMAL(10,2) NOT NULL,
+      "imageUrl" TEXT NULL,
+      "inStock" BOOLEAN NOT NULL DEFAULT false,
+      category TEXT NOT NULL
+    );
+      CREATE TABLE users(
+        id SERIAL PRIMARY KEY,
+        "firstName" VARCHAR(255) NOT NULL,
+        "lastName" VARCHAR(255) NOT NULL,
+        email VARCHAR(255) UNIQUE NOT NULL,
+        "imageUrl" TEXT NULL,
+        username VARCHAR(255) UNIQUE NOT NULL,
+        password VARCHAR(255) UNIQUE NOT NULL,
+        "isAdmin" BOOLEAN NOT NULL DEFAULT false
+      );
+      CREATE TABLE orders(
+        id SERIAL PRIMARY KEY,
+        status VARCHAR(255) DEFAULT 'created',
+        "userId" INTEGER REFERENCES users(id),
+        "datePlaced" DATE 
+      );
+      CREATE TABLE order_products(
+        id SERIAL PRIMARY KEY,
+        "productId" INTEGER REFERENCES products(id),
+        "orderId" INTEGER REFERENCES orders(id),
+        price DECIMAL(10,2) NOT NULL ,
```

Comment:
Just so you know, decimals get a bit hairy in javascript for prices.  SOME times... people choose to store "prices" as pennies, and convert to dollars on the frontend or elsewhere.